torch/_subclasses/fake_tensor.py

```diff
@@ -843,9 +843,7 @@ def wrap(e, device=None):
             return tree_map(partial(wrap), r)
 
     def may_turn_const(self, t):
-        return (
-            self.const_tensors and t.numel() <= CONSTANT_NUMEL_LIMIT and not t.is_sparse
-        )
+        return self.const_tensors and t.numel() <= CONSTANT_NUMEL_LIMIT and not t.is_sparse and not isinstance(t, FakeTensor)
```

Comment:
Why is this?

Reply:
you don't want to make a constant when you run an fx graph, and the input is a fake tensor (obviously that's not a constant)